iot-hub/Tutorials/ConnectivityTests/SimulatedDevice-3.js

```diff
@@ -82,6 +82,16 @@ client.open(function (err) {
       var data = JSON.stringify({ temperature: temperature, humidity: humidity });
       var message = new Message(data);
 
+      // Add the level
+      var level = Math.random();
+      if(level >= 0.7){
+        message.properties.level="critical"
```

Comment:
I don't think this is a valid use of the properties` object of the `Message` class. see https://github.com/Azure/azure-iot-sdk-node/blob/master/common/core/src/properties.ts for details. I'd be surprised if this worked.

Also, as a nitpick: spacing of braces/curly braces is inconsistent with other samples.